sknpm/src/main.sk

```diff
@@ -885,25 +885,20 @@ fun copyAndConvertImport(
     print_string(">> Copy js files and convert to valid imports.");
   };
   nameLine = "var wasm64 = \"/*--NAME--*/\";";
-  modulesLines = "var modules = [ /*--MODULES--*/];";
+  modulesLines = "/*--MODULES--*/";
   base64Line = "export var base64 = \"\";";
   versionLine = "const npmVersion = \"\";";
-  extensionsLine = "/*--EXTENSIONS--*/";
```

Comment:
Ah, cool. I considered doing this, but wasn't sure what it was intended for. I figured best to leave it alone. 🤣